src/screens/Login/index.jsx

```diff
@@ -1,10 +1,63 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { setCookie } from 'utils/CookieUtils';
 import * as S from './styles';
 
-const Login = () => (
-  <S.Wrap>
-    <S.Title>Login</S.Title>
-  </S.Wrap>
-);
+const Login = () => {
+  const [loginData, setLoginData] = useState({
+    email: '',
+    password: ''
+  });
+  const [error, setError] = useState('');
+
+  const handleDataChange = (event) => {
+    const { target } = event;
+    setLoginData((prev) => ({ ...prev, [target.name]: target.value }));
+  };
+
+  const handleLogin = async (event) => {
+    try {
+      event.preventDefault();
+
+      const response = await fetch(
+        'https://nestbe.herokuapp.com/api/user/login',
+        {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json;charset=utf-8'
+          },
+          body: JSON.stringify(loginData)
+        }
+      );
```

Comment:
Should create API service

https://github.com/aspasionov/pet-project-todo/pull/1/files#diff-2f522abeeab5a77415c12f18967c2490daed674550eeeae3daa538ef09879c6d